tests/restapi/test_routes.py

```diff
@@ -1029,6 +1029,21 @@ def test_structure_download_false(self):
         structure_data = load_node(node_uuid)._exportcontent('xsf')[0]  # pylint: disable=protected-access
         assert response['data']['download']['data'] == structure_data.decode('utf-8')
 
+    def test_structure_download_false_lowercase(self):
```

Comment:
```suggestion
    @pytest.mark.parametrize("download", ["false", "False"])
    def test_structure_download_false(self, download):
    ...
        url = f'{self.get_url_prefix()}/nodes/{node_uuid}/download?download_format=xsf&download={download}'
    ...
```
I think this is where you can use parametrize and more concise?

Reply:
perfect, thanks!